### DISCUSSION TITLE
Relating to issue 14925 - Dovecot IMAP

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15161] [Feature improvement]
https://github.com/YetiForceCompany/YetiForceCRM/issues/14925
In above issue is nice answer to how to set save folder in SMTP Configuration.
Now, the thing is, mail server is on standard postfix/dovecot. What would be a naming convention of folders I want to save sent emails in case of dovecot IMAP?

**Comment by s2b-git** [comment]
Workaround is to BCC to 'user'+sent@domain.tld, but if anyone could answer I will appreciate :-)

**Reply by blu-IT** [reply]
I am not sure if I understand your question. As far as I know (unfortunately) there is no naming convention, as every IMAP client has its own behavior of naming folders. So if your mailbox is only fetched by YF and you assigned the correct folders to the YF mailscanner it has no impact how you name the folders.
